### PR TITLE
Added Menu Patch to Auto Totem

### PR DESCRIPTION
**Pull request by allbombson**
This allows AutoTotem to work when your inv is open or your alt tabbed, or if any gui is open for that matter, Just like future.

### CODE REVIEW

File: src/main/java/com/matt/forgehax/mods/AutoTotemMod.java [hunk 1]
```diff
@@ -21,7 +22,17 @@
   public AutoTotemMod() {
     super(Category.COMBAT, "AutoTotem", false, "Automatically move totems to off-hand");
   }
-  
+
+  private final Setting<Boolean> MenuPatch =
```
Comment (fr1kin) [inline src/main/java/com/matt/forgehax/mods/AutoTotemMod.java]: rename MenuPatch to something more appropriate like allowGui

File: src/main/java/com/matt/forgehax/mods/AutoTotemMod.java [hunk 2]
```diff
@@ -21,7 +22,17 @@
...
+      getCommandStub()
+          .builders()
+          .<Boolean>newSettingBuilder()
+          .name("MenuPatch")
```
Comment (fr1kin) [inline src/main/java/com/matt/forgehax/mods/AutoTotemMod.java]: you should use camel case or underscores for field names, but the `.name` should not be capitalized and underscores/whitespace should be hyphed (ex: `.name("allow-gui")`)

File: src/main/java/com/matt/forgehax/mods/AutoTotemMod.java [hunk 3]
```diff
@@ -21,7 +22,17 @@
...
+          .description(
+              "Menu Patch - Lets AutoTotem work in menus.")
```
Comment (fr1kin) [inline src/main/java/com/matt/forgehax/mods/AutoTotemMod.java]: `Menu Patch - ` should be cut out

Comment (allbombson) [reply src/main/java/com/matt/forgehax/mods/AutoTotemMod.java]: ok changes will be pushed asap

**Comment by allbombson** [comment]
Changes have been made @fr1kin

**Comment by allbombson** [comment]
HAHA YES, Gamer Time. I have contributed to a repo.

**Pull request merged**